test/jest.setup.ts

```diff
@@ -4,7 +4,7 @@ require('ts-node/register')
 import fs from 'fs'
 import path from 'path'
 
-const dir = path.join(__dirname, '..', '.test-data')
+const dir = path.join(__dirname, '.test-data')
```

Comment:
I changed the folder of the test-data to be in /test/.test-data instead of at the root of the repo. You could see the files move on the left whenever .test-data was created/deleted which was a bit annoying